build_cmake/strict/cl_strict.cmake

```diff
@@ -0,0 +1,35 @@
+#
+# Public Domain 2014-present MongoDB, Inc.
+# Public Domain 2008-2014 WiredTiger, Inc.
+#  All rights reserved.
+#
+# See the file LICENSE for redistribution information.
+#
+
+# Warning level 3.
+list(APPEND win_c_flags "/WX")
+# Ignore warning about mismatched const qualifiers.
+list(APPEND win_c_flags "/wd4090")
```

Comment:
Were these ignores taken directly from the sconscript?

Reply:
Yep, most of these flags have been adapted from the sconscript

Reply:
Can I be annoying and ask you to check if we still need the warning ignores?

Reply:
Yep, I tried turning off the compiler warnings. The warnings unfortunately still occur so they are still needed if compiling with strict.